app/src/main/java/com/hedvig/app/feature/offer/ui/OfferActivity.kt

```diff
@@ -262,11 +263,17 @@ class OfferActivity : BaseActivity(R.layout.activity_offer) {
     }
 
     private fun inflateMenu(loginStatus: LoginStatus) {
-        binding.offerToolbar.menu.clear()
+        val menu = binding.offerToolbar.menu
+        menu.clear()
         when (loginStatus) {
             LoginStatus.ONBOARDING,
             LoginStatus.IN_OFFER -> binding.offerToolbar.inflateMenu(R.menu.offer_menu)
-            LoginStatus.LOGGED_IN -> binding.offerToolbar.inflateMenu(R.menu.offer_menu_logged_in)
+            LoginStatus.LOGGED_IN -> {
+                binding.offerToolbar.inflateMenu(R.menu.offer_menu_logged_in)
+                menu.getItem(0).actionView.setOnClickListener {
```

Comment:
Does this not happen automatically? 😮

Reply:
I think its some weird behaviour because we are using an action view for the item. We are having the same behaviour in LoggedInActivity.

Reply:
Ah, makes sense!